src/libs/ReportUtils.ts

```diff
@@ -1823,44 +1835,58 @@ function canEditMoneyRequest(reportAction: OnyxEntry<ReportAction>, fieldToEdit
     }
 
     const moneyRequestReport = getReport(String(moneyRequestReportID));
-    const isReportSettled = isSettled(moneyRequestReport?.reportID);
-    const isApproved = isReportApproved(moneyRequestReport);
-    const isAdmin = isExpenseReport(moneyRequestReport) && (getPolicy(moneyRequestReport?.policyID ?? '')?.role ?? '') === CONST.POLICY.ROLE.ADMIN;
+    const policy = getPolicy(moneyRequestReport?.policyID ?? '');
+    const isAdmin = isExpenseReport(moneyRequestReport) && policy.role === CONST.POLICY.ROLE.ADMIN;
+    const isManager = isExpenseReport(moneyRequestReport) && currentUserAccountID === moneyRequestReport?.managerID;
     const isRequestor = currentUserAccountID === reportAction?.actorAccountID;
 
-    if (isAdmin && !isRequestor && fieldToEdit === CONST.EDIT_REQUEST_FIELD.RECEIPT) {
-        return false;
+    // Admin & managers can always allow coding fields such as tag, category, billable, etc. As long as the report has a state higher than OPEN.
+    if ((isAdmin || isManager) && !isOpenReport(moneyRequestReport)) {
+        return true;
     }
 
-    if (isAdmin) {
-        return true;
+    // We allow editing of PROCESSING reports in Free policies, but not in paid policie (yet)
+    if (policy.type === CONST.POLICY.TYPE.FREE) {
+        return isRequestor && isProcessingReport(moneyRequestReport);
     }
 
-    return !isApproved && !isReportSettled && isRequestor;
+    return !isReportApproved(moneyRequestReport) && !isSettled(moneyRequestReport?.reportID) && isRequestor;
 }
 
-/**
- * Checks if the current user can edit the provided property of a money request
- *
- */
-function canEditFieldOfMoneyRequest(reportAction: OnyxEntry<ReportAction>, reportID: string, fieldToEdit: ValueOf<typeof CONST.EDIT_REQUEST_FIELD>): boolean {
+function canEditFieldOfMoneyRequest(reportAction: OnyxEntry<ReportAction>, transaction: Transaction, fieldToEdit: ValueOf<typeof CONST.EDIT_REQUEST_FIELD>): boolean {
```

Comment:
[Here is a use](https://github.com/Expensify/App/blob/8cb48e468ce3c09ada2d58aaae5354ec206104f9/src/pages/iou/IOUCurrencySelection.js#L89) of `canEditFieldOfMoneyRequest`. We need to change the report id param passed here to transaction.

Reply:
Good catch, will update. Thanks!